pkgs/applications/audio/spotify/default.nix

```diff
@@ -78,11 +79,10 @@ stdenv.mkDerivation {
   # https://community.spotify.com/t5/Desktop-Linux/Redistribute-Spotify-on-Linux-Distributions/td-p/1695334
   src = fetchurl {
     url = "https://api.snapcraft.io/api/v1/snaps/download/pOBIoZ2LrCB3rDohMxoYGnbN14EHOgD7_${rev}.snap";
-    sha512 = "5b3d5d1f52a554c8e775b8aed16ef84e96bf3b61a2b53266e10d3c47e341899310af13cc8513b04424fc14532e36543a6fae677f80a036e3f51c75166d8d53d1";
+    sha512 = "5d61a2d5b26be651620ab5d18d3a204d8d7b09dcec8a733ddc176c44cb43e9176c4350933ebe4498b065ba219113f3226c13bea9659da738fe635f41d01db303";
   };
 
-  nativeBuildInputs = [ makeWrapper ];
-  buildInputs = [ squashfsTools ];
+  nativeBuildInputs = [ makeWrapper squashfsTools ];
 
   dontStrip = true;
```

Comment:
Does stripping break it?

Reply:
I'm not sure. This was just the suggestion in @r-rmcgibbo's comment.

Reply:
Oops, I just realized you commented on line 87, not 84-85... That's a great question re `dontStrip`. That decision was before my time... Perhaps one of the maintainers would know?

Reply:
Good catch. This was already included in the original commit (f81bc24790895bb06e3ead3d684042e94fc2c87a). I'd say there is a good chance that it is not needed (anymore). I think we should generally not require unrelated improvements on update PRs though.

Reply:
Yeah, I'd be happy to submit a follow-up PR removing `dontStrip`

Reply:
So we don't need it?

```suggestion
```